index.html

```diff
@@ -57,15 +68,20 @@
 				<source media="(min-width: 58rem)" srcset="./assets/2.png" width='2000' height='1243' />
 				<img class="layer-2" src="./assets/2.png" alt="" width='2000' height='1243' />
 			</picture>
+      <picture>
+				<source media="(max-width: 58rem)" srcset="./assets/blank.svg" width="1" height="1" />
+				<source media="(min-width: 58rem)" srcset="./assets/3.png" width='2000' height='1243' />
+				<img class="layer-3" src="./assets/3.png" alt="" width='2000' height='1243' />
+			</picture>
 		</div>
 
 		<div class="hero-container">
 			<div class="hero-content">
 				<h1>Step into the<br/>Pluriverse</h1>
 				<p>
 					Read the guide &nbsp;
-					<a href="/pdf/guide.pdf">[PDF] &nbsp;</a>
-					<a href="https://online.fliphtml5.com/wryqh/bdyf/"> [Flipbook]</a>
+					<a href="/pdf/pluriverse-guide.pdf">[PDF] &nbsp;</a>
```

Comment:
@cristinalare heads up the site 404s now when u click on the pdf option

Reply:
https://pluriverse.wtf/

Reply:
yes, right, /pdf is in gitignore. im sending you the pdf rn